### PR TITLE
Governance.md

### PR DESCRIPTION
**Pull request by tgreer**
[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/202)

### CODE REVIEW

**Comment by tracernz** [comment]
:lgtm_strong: Thanks for all your hard work on this and the election @tgreer.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/202#-:-K54txK0zA_eQWX98YOV:353488892)*

**Comment by tgreer** [comment]
This resolves #43

File: Governance.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+==========
+On 8th December 2015, 17 members of the active community voted to elect the first dRonin Executive Committee.
+
+Those voted in were: hshred, icee, tracernz.
+
+They will now servce the project for a 6 month period.
+
+Executive Team's Role
+=====================
+* Guide the project forwards.
```
Comment (readerror67) [inline Governance.md]: s/forwards/forward

File: Governance.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Governance
+==========
+On 8th December 2015, 17 members of the active community voted to elect the first dRonin Executive Committee.
+
+Those voted in were: hshred, icee, tracernz.
+
+They will now servce the project for a 6 month period.
```
Comment (tracernz) [inline Governance.md]: s/servce/service/

Comment (readerror67) [inline Governance.md]: Wouldn't it be serve ?

Comment (tracernz) [inline Governance.md]: Yeah

File: Governance.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Governance
+==========
+On 8th December 2015, 17 members of the active community voted to elect the first dRonin Executive Committee.
+
+Those voted in were: hshred, icee, tracernz.
```
Comment (mluessi) [inline Governance.md]: Let's use github user names or real names. Also, should we call it Executive Board? Sounds more professional (committee also sounds like it won't get anything done ;)

Comment (kubark42) [inline Governance.md]: Congrats to all!

P.S. Agreed about real names.

File: Governance.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Governance
+==========
+On 8th December 2015, 17 members of the active community voted to elect the first dRonin Executive Committee.
+
+Those voted in were: 
+* Martin Luessi (hshred)
+* Michael Lyle (icee)
+* Michael Corcoran (tracernz)
```
Comment (mluessi) [inline Governance.md]: Can you link the names to the github accounts and put (IRC: hshred etc)?

**Pull request closed without merging**